### PR TITLE
Components

### PR DESCRIPTION
**Pull request by marurevi**
## Math magicians: components
I implement some components to build the calculator. Here some changes:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:

<p align="center">
  Calculator
</p>

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="calculator">
+      <Calculator />
     </div>
   );
 }
```
Comment (codepantha) [inline src/App.js:9]: - [ ] According to project requirements, please ensure to make your `<App>` component class-based. It is currently a functional component :thumb

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class Calculator extends React.PureComponent {
+  render() {
+    return (
+      <ul className="grid-container">
+        <li className="fullspan"><div className="output">0</div></li>
+
+        <li><button className="btn-grey" type="button">AC</button></li>
+        <li><button className="btn-grey" type="button">+/-</button></li>
+        <li><button className="btn-grey" type="button">%</button></li>
+        <li><button className="btn-orange" type="button">/</button></li>
```
Comment (codepantha) [inline src/components/calculator.js:13]: - [ ] [OPTIONAL] You can make your app even better by creating a reusable `<Button>` component 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @marurevi 👋, 

## STATUS: REQUESTED CHANGES ♻️ 

Good job so far. Your design is beautiful!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] showing users how to clone your application to their local machine in your README file. You could use the line below 👇 
```
https://github.com/marurevi/Math-Magicians.git
```

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,19 @@
...
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <Calculator />
+      </div>
+    );
+  }
```
Comment (mateo951) [inline src/App.js:16]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @marurevi, 

Well done making the changes requested by the previous reviewer ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it  🙌
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**